testing/integration/kubernetes_agent_standalone_test.go

```diff
@@ -266,7 +266,10 @@ func TestKubernetesAgentHelm(t *testing.T) {
 		steps      []k8sTestStep
 	}{
 		{
-			name: "helm standalone agent default kubernetes privileged",
+			// Configure the perNode and clusterWide agents to both use host networking. On the node that
+			// runs the clusterWide agent, this tests that two agents do not try to bind to the same
+			// gRPC control protocol port by default preventing one from starting.
+			name: "helm standalone agent default kubernetes privileged without host network port collision",
```

Comment:
@pkoutsovasilis I just added host networking to the clusterWide agent in an existing k8s test to test this, let me know if it would be better as a separate test.

Reply:
yy that makes total sense, I do also believe that there is no need to have a separate one just for this case 🙂 (our k8s integration tests already take a lot of time)